Content.Server/Materials/MaterialComponent.cs

```diff
@@ -18,13 +13,11 @@ namespace Content.Server.Materials
     public sealed class MaterialComponent : Component
     {
         [ViewVariables]
-        [DataField("materials", customTypeSerializer:typeof(PrototypeIdListSerializer<MaterialPrototype>))]
+        [DataField("materials", customTypeSerializer:typeof(PrototypeIdDictionarySerializer<int, MaterialPrototype>))]
         // ReSharper disable once CollectionNeverUpdated.Local
-        private readonly List<string> _materials = new();
-        public IEnumerable<string> MaterialIds => _materials;
-        
-        [DataField("volume")]
-        public int volume = 100;
+        public readonly Dictionary<string, int> _materials = new();
+        public List<string> MaterialIds => _materials.Keys.ToList();
+        public List<int> MaterialVolume => _materials.Values.ToList();
```

Comment:
Unused?

Reply:
Looks like the Volume one is unused, but the MertialIds seems to be used below it with the method that gathers all the materials something is made of (it was there before I touched MaterialComponent lol) 

I could just remove the MaterialVolume one?